diff --git a/.gitignore b/.gitignore
index 10b21db..f34cfee 100644
--- a/.gitignore
+++ b/.gitignore
@@ -6,4 +6,4 @@ venv/
 .venv/
 env/
 **/venv/
-**/.venv/
\ No newline at end of file
+
diff --git a/Offline 2 - Max cut by Grasp/plot_maxcut.py b/Offline 2 - Max cut by Grasp/plot_maxcut.py
index 729291d..59bcd9c 100644
--- a/Offline 2 - Max cut by Grasp/plot_maxcut.py	
+++ b/Offline 2 - Max cut by Grasp/plot_maxcut.py	
@@ -30,7 +30,7 @@ plt.bar([i + 2*bar_width for i in x], local, width=bar_width, label='Local Searc
 plt.xticks(x, graphs)
 plt.xlabel('Graph')
 plt.ylabel('Max Cut Value')
-plt.title('Max Cut (Graph 1–10)')
+plt.title('Max Cut (Graph 1-10)')
 plt.legend()
 plt.grid(axis='y', linestyle='--', alpha=0.7)
 plt.tight_layout()
